dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -633,23 +662,23 @@ protected void loadRelationshipFields(Contentlet con, Map<String,String> m) thro
 
         for(Map<String, Object> relatedEntry : db.loadObjectResults()) {
 
-            String childId = relatedEntry.get("child").toString();
-            String parentId = relatedEntry.get("parent").toString();
-            String relType=relatedEntry.get("relation_type").toString();
-            String order = relatedEntry.get("tree_order").toString();
+            String childId = relatedEntry.get(ESMappingConstants.CHILD).toString();
+            String parentId = relatedEntry.get(ESMappingConstants.PARENT).toString();
+            String relType=relatedEntry.get(ESMappingConstants.RELATION_TYPE).toString();
+            String order = relatedEntry.get(ESMappingConstants.TREE_ORDER).toString();
 
             Relationship rel = FactoryLocator.getRelationshipFactory().byTypeValue(relType);
 
             if(rel!=null && InodeUtils.isSet(rel.getInode())) {
                 boolean isSameStructRelationship = rel.getParentStructureInode().equalsIgnoreCase(rel.getChildStructureInode());
 
                 String propName = isSameStructRelationship ?
-                        (con.getIdentifier().equals(parentId)?rel.getRelationTypeValue() + "-child":rel.getRelationTypeValue() + "-parent")
+                        (con.getIdentifier().equals(parentId)?rel.getRelationTypeValue() + ESMappingConstants.SUFFIX_CHILD:rel.getRelationTypeValue() + ESMappingConstants.SUFFIX_PARENT)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 185 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)